lib/chef/provisioning/docker_driver/docker_transport.rb

```diff
@@ -236,40 +150,14 @@ def using_boot2docker?
       end
     end
 
-    # Copy of container.attach with timeout support and pipeline
-    def attach_with_timeout(container, read_timeout, options = {}, &block)
-      opts = {
-        :stream => true, :stdout => true, :stderr => true
-      }.merge(options)
-      # Creates list to store stdout and stderr messages
-      msgs = Docker::Messages.new
-      connection.start_request(
-        :post,
-        "/containers/#{container.id}/attach",
-        opts,
-        :response_block => attach_for(block, msgs),
-        :read_timeout => read_timeout,
-        :pipeline => true,
-        :persistent => true
+    def container_path(path)
+      docker_root = File.join(
+        Docker.info['DockerRootDir'],
+        Docker.info['Driver'],
+        'mnt'
       )
```

Comment:
@marc- I need to test on centos but on a default ubuntu setup, `Docker.info['Driver']` gets `aufs`.